sdks/java/container/build.gradle

```diff
@@ -86,51 +76,21 @@ licenseReport {
   renderers = [new JsonReportRenderer()]
 }
 
-def imageJavaVersion = project.hasProperty('imageJavaVersion') ? project.findProperty('imageJavaVersion') : '8'
-docker {
```

Comment:
Maybe we should keep the `:sdks:java:container:docker` task for a bit and just have it error with a message directing people to java8:docker and/or java11:docker?

Reply:
My current approach was just to just add a `task docker { throw ... }` block to replace the Docker plugin-created tasks, but but I can't figure out how to get the license report to ignore this task (and avoid throwing the error). This might just be a gradle question but a) should I be throwing the error differently and b) should the license report be the same task for both java8/11?

Reply:
Hm ok, I'm not sure if we have any precedent to follow for this. I don't think it's worth going to a lot of trouble for. Instead you could just send an FYI email to dev@ once this is merged letting people know they may need to change :sdks:java:container:docker -> :sdks:java:container:java8:docker.